app/config/version.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+import subprocess
+
+# Note that this file is overwritten during the build
+# process by either the git tag of the commit
+# (see Dockerfile for details)
+
+proc = subprocess.Popen(["git", "describe", "--tags"],
+                        stdout=subprocess.PIPE,
+                        stderr=subprocess.PIPE)
+stdout, stderr = proc.communicate()
+git_version = stdout.decode('utf-8').strip()  # pylint: disable=invalid-name
```

Comment:
Disabling `invalid-name` is really required here ? which name is triggering this pylint issue ? Maybe we could add this name in the pylint good name list, for me all names in this line are really valid.

Reply:
I didn't understand this either, for some reason it expected an UPPERCASE var name...

Reply:
Ok I understand now, missed that. The python guideline expect to have global constant as UPPERCASE :-) and this case it is a constant, so maybe change it to uppercase this way your 100% compliant with our guideline ;-)

Reply:
Ok